sympy/physics/control/tests/test_lti.py

```diff
@@ -1705,6 +1706,30 @@ def test_conversion():
     raises(ValueError, lambda: TransferFunction(b*s**2 + p**2 - a*p + s, b - p**2, s).rewrite(StateSpace))
 
 
+def test_StateSpace_state_output_vector():
+    # https://web.mit.edu/2.14/www/Handouts/StateSpaceResponse.pdf
+    # https://lpsa.swarthmore.edu/Transient/TransMethSS.html
+    A1 = Matrix([[0, 1], [-2, -3]])
+    B1 = Matrix([[0], [1]])
+    C1 = Matrix([[1, -1]])
+    D1 = Matrix([0])
+    i1 = Matrix([[1], [2]])
+    t = symbols('t')
+    ss1 = StateSpace(A1, B1, C1, D1)
+    assert ss1.state_vector(initial_conditions=i1, var=t) == Matrix([[ 4*exp(-t) - 3*exp(-2*t)],
+                                                              [-4*exp(-t) + 6*exp(-2*t)]])
```

Comment:
There are no tests with symbolic matrices. Will this work with such matrices?

Reply:
I don't think the `StateSpace` model supports symbolic matrices as of now. Kindly raise an issue and we can discuss it there.

Reply:
It may be possible that some functions in `StateSpace` model doesn't support *symbolic matrices* but I've tested that most of the methods in `StateSpace` support symbolic.

```
>>> a11, a12, a21, a22 = symbols('a11 a12 a21 a22')
>>> b1, b2 = symbols('b1 b2')
>>> c1, c2 = symbols('c1 c2')
>>> d1 = symbols('d1')
>>> A = Matrix([[a11, a12], [a21, a22]])
>>> B = Matrix([[b1], [b2]])
>>> C = Matrix([[c1, c2]])
>>> D = Matrix([d1])
>>> ss = StateSpace(A, B, C, D)
>>> ss
StateSpace(
Matrix([
[a11, a12],
[a21, a22]]),

Matrix([
[b1],
[b2]]),

Matrix([[c1, c2]]),

Matrix([[d1]]))
>>> ss.rewrite(TransferFunction)
[[TransferFunction(-c1*(a12*b2 - a22*b1 + b1*s) - c2*(-a11*b2 + a21*b1 + b2*s) + d1*(-a11*a22 + a11*s + a12*a21 + a22*s - s**2), -a11*a22 + a11*s + a12*a21 + a22*s - s**2, s)]]
```

Reply:
Ohh yes, I recall this. Thanks for letting me know.